mock-mhs-adaptor/src/main/java/uk/nhs/adaptors/mockmhsservice/service/MockMhsService.java

```diff
@@ -30,60 +29,65 @@
 @Slf4j
 @RequiredArgsConstructor(onConstructor = @__(@Autowired))
 public class MockMhsService {
-
     private final InboundProducer inboundProducer;
     private final ObjectMapper objectMapper = new ObjectMapper();
     private final HttpHeaders headers = new HttpHeaders();
-    private final String mockValidInteractionId = "RCMR_IN030000UK06";
 
-    private final InputStream inputStream = this.getClass().getClassLoader().getResourceAsStream("COPC_IN000001UK01.json");
-    private final String stubInboundMessage = IOUtils.toString(inputStream, StandardCharsets.UTF_8);
-    private final InputStream inputStream2 = this.getClass().getClassLoader().getResourceAsStream("StubEbXmlResponse.xml");
-    private final String stubEbXmlResponse = IOUtils.toString(inputStream2, StandardCharsets.UTF_8);
-    private final InputStream inputStream3 = this.getClass().getClassLoader().getResourceAsStream("InternalServerError.html");
-    private final String internalServerErrorResponse = IOUtils.toString(inputStream3, StandardCharsets.UTF_8);
+    private static final String EXTRACT_CORE_INTERACTION_ID = "RCMR_IN030000UK06";
+    private static final String ACKNOWLEDGEMENT_INTERACTION_ID = "MCCI_IN010000UK13";
+    private static final ClassLoader CLASS_LOADER = MockMhsService.class.getClassLoader();
```

Comment:
Could give it a little bit less generic name? Like "MOCK_MHS_SERVICE_CLASS_LOADER"?